management/nodes/startupScripts/win2016-20170307-internal.ps1

```diff
@@ -0,0 +1,64 @@
+Set-ExecutionPolicy Unrestricted
+
+$machineSetupLog = 'C:\setup.log'
+$jenkinsServerUrl = $args[0]
+echo ('Jenkins Server URL: {0}' -f $jenkinsServerUrl) | Out-File -Append $machineSetupLog
+$vmName = $args[1]
+echo ('Virtual Machine Name: {0}' -f $vmName) | Out-File -Append $machineSetupLog
+$secret = $args[2]
+echo ('Secret: {0}' -f $secret) | Out-File -Append $machineSetupLog
+
+$winLogonRegistryPath = 'hklm:\software\microsoft\windows nt\currentversion\winlogon\'
+$jenkinsDirectory = 'C:\Jenkins'
+$jenkinsLaunchCmd = Join-Path -Path $jenkinsDirectory -ChildPath 'launch.cmd'
+$jenkinsStartupScript = Join-Path -Path $jenkinsDirectory -ChildPath 'jenkins-windows-startup.ps1'
+$jenkinsLogFile = Join-Path -Path $jenkinsDirectory -ChildPath 'log.txt'
+# The image needs to have LUA off to run properly. If you want the VM to have administrative permissions for
```

Comment:
This comment and the `$enableLUA = ...` line seems to be at odds with each other.